protobuf/model_config.proto

```diff
@@ -1869,6 +1869,84 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+//@@
+//@@  .. cpp:var:: message ModelMetrics
+//@@
+//@@     The metrics setting of this model. Consider reusing this message body
+//@@     for backend metric custom configuration.
```

Comment:
>`Consider reusing this message body for backend metric custom configuration.` 

This looks more like an internal note or TODO than public-facing documentation. Can probably be removed.

Reply:
I added this note per discussion with Guan. What's the best place to properly document the note?

Reply:
It's fine to leave here, but would generally add something easy to grep or notice later when reading, such as `NOTE: ...`